### PR TITLE
Add one More BlockExlporer and improve the Documetation

### PR DESCRIPTION
**Pull request by hardiktheprogrammer**
- Add Subscan BlockExlporer 
- Added in Table 
-  Improve the Documentation

### CODE REVIEW

**Comment by hardiktheprogrammer** [comment]
@ImmaZoni  can you please review and merge this PR

File: docs/develop/nova/block_explorer.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
----
-title: Subspace block explorer
-sidebar_position: 10
-description: Subspace hosted block explorer
-keywords:
-  - block explorer
-  - subspace network
----
+# Subspace block explorer
```
Comment (EmilFattakhov) [inline docs/develop/nova/block_explorer.md:1]: Let's keep the section at the top unchanged as it helps with navigation and SEO. Thanks!

File: docs/develop/nova/block_explorer.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+On the top of the page, you can easily toggle between *all available networks and EVM.*
+
+The following block explorers provide information on Subspace .
+
+
+
+| Block Explorer   name | URL |
+|------------------------------|----------------------------------------|
+| Astral  | [explorer.subspace.network](https://explorer.subspace.network/#/gemini-3g/consensus)   |
+| Subscan  | [subspace.subscan](https://subspace.subscan.io/) |
```
Comment (EmilFattakhov) [inline docs/develop/nova/block_explorer.md:16]: I suggest leaving the image of Astral (Subspace Explorer) right after the table, as it's a preferred tool to view the data. 

BlockExplorer-1

**Review by EmilFattakhov — CHANGES_REQUESTED** [review]
Thanks for your suggestions @hardiktheprogrammer!
That's a good idea to link on the page as well. Left a few comments, should be a fairly small fix!

File: docs/develop/nova/block_explorer.md (hunk 2 above)
Comment (hardiktheprogrammer) [reply docs/develop/nova/block_explorer.md:16]: should i also add subscan image also

File: docs/develop/nova/block_explorer.md (hunk 1 above)
Comment (hardiktheprogrammer) [reply docs/develop/nova/block_explorer.md:1]: okkay cool

File: docs/develop/nova/block_explorer.md (hunk 2 above)
Comment (EmilFattakhov) [reply docs/develop/nova/block_explorer.md:16]: Sure, let's add one for Subscan as well. I guess in this case, we can omit to put the resources in a table, and just put them one after another with one screenshot for each. Thanks!

Comment (hardiktheprogrammer) [reply docs/develop/nova/block_explorer.md:16]: okkaay

**Comment by EmilFattakhov** [comment]
@hardiktheprogrammer how're the updates going? We're nearly there, we'd really appreciate it if you could make some final adjustments :)

**Comment by hardiktheprogrammer** [comment]
> @hardiktheprogrammer how're the updates going? We're nearly there, we'd really appreciate it if you could make some final adjustments :)

Sorry for the late i was a little bit busy with the starkness technical workshops , right Now going  to add the image

**Comment by EmilFattakhov** [comment]
@hardiktheprogrammer are you still working on the changes? We will keep the PR open for another 7 days and close it afterwards, thanks for your understanding!

**Comment by EmilFattakhov** [comment]
Closed due to inactivity.

**Pull request closed without merging**